pkg/template/controller/templateinstance_controller.go

```diff
@@ -501,7 +501,7 @@ func (c *TemplateInstanceController) instantiate(templateInstance *templatev1.Te
 			}
 		}
 		if createErr != nil {
-			allErrors = append(allErrors, mappingErr)
+			allErrors = append(allErrors, createErr)
```

Comment:
must have been a copy and paste from above ...

Reply:
yeah, seems like.